### ISSUE TITLE
sqlite3: command not found

### ISSUE THREAD

**Issue by StormPooper** [TheCaptain989/lidarr-flac2mp3#26]
Noticed recently that I get the following error in my event log in Lidarr:

```yaml
/usr/local/bin/flac2mp3.sh: line 32: sqlite3: command not found
```

It doesn't stop the conversion from happening, so not a big deal, but thought I'd bring it up.

Looking at the script, I wonder if it the setup script needs to check for the dependency and install it if missing. I tried recreating my Lidarr image with the plugin but it made no difference.

**Comment by TheCaptain989** [comment]
This would be the check for the Recycle Bin failing.  It won't affect the overall script function, but your FLAC files will be permanently deleted.
What version of lidarr are you running?

**Comment by StormPooper** [comment]
Ah, that'd make sense as to why it's still converting the MP3s. It might be worth noting that I don't have the Recycle Bin path set. Here's the version info:

```
Version
    0.8.1.2135
Package Version
    0.8.1.2135-ls49 by linuxserver.io 
```

**Comment by TheCaptain989** [comment]
Thank you.  At the very least I'll make the script more forgiving to this error, but ideally I'll use the API to get the Recycle Bin instead of a direct DB query.  The Lidarr team must have updated something.
Thanks for reporting this!

**Comment by StormPooper** [comment]
Sounds good, thanks for replying and taking a look at it 👍

**Comment by TheCaptain989** [comment]
Please check that this is now fixed.  Use `DOCKER_MODS=thecaptain989/lidarr-flac2mp3:latest` to pull the dev instance.

**Comment by StormPooper** [comment]
Can confirm that the error message is now gone, thanks for the quick response and update!

**Comment by TheCaptain989** [comment]
@StormPooper, I'll try to publish this this week (create a PR for linuxserver.io team).

**Comment by TheCaptain989** [comment]
The team published my changes with [PR #211](https://github.com/linuxserver/docker-mods/pull/211)
